src/js/actions/AppsActions.js

```diff
@@ -92,21 +92,21 @@ var AppsActions = {
       headers: {
         "Content-Type": "application/json"
       },
-      url: config.apiURL + "v2/apps/" + appId + "/restart"
-    })
-    .success(function (app) {
-      AppDispatcher.dispatch({
-        actionType: AppsEvents.RESTART_APP,
-        data: app,
-        appId: appId
-      });
+      url: `${config.apiURL}v2/apps/${appId}/restart`
```

Comment:
This doesn't necessarily block this PR since it's a pre-existing issue and doesn't actually cause malfunctions, but since each `appId` has a preceding forward slash, this will cause duplicated forward slashes eg `v2/apps//foobar/restart`